src/components/POLine/Eresources/EresourcesForm.js

```diff
@@ -22,6 +22,10 @@ import { Required } from '../../Utils/Validate';
 import InventoryRecordTypeSelectField from '../../../settings/InventoryRecordTypeSelectField';
 import css from './EresourcesForm.css';
 
+const ACCESS_PROVIDER_TETHER_CONFIG = {
+  attachment: 'middle center',
```

Comment:
whaaaaat?) looks like magic)

Reply:
magic from `react-tether` that uses `tether` :)